view/next-project/src/pages/year_periods/index.tsx

```diff
@@ -0,0 +1,181 @@
+import clsx from 'clsx';
+import Head from 'next/head';
+import { useRouter } from 'next/router';
+import { useEffect, useState, useMemo } from 'react';
+import { useRecoilValue } from 'recoil';
+
+import OpenAddModalButton from '@/components/year_periods/OpenAddModalButton';
+import OpenDeleteModalButton from '@/components/year_periods/OpenDeleteModalButton';
+import OpenEditModalButton from '@/components/year_periods/OpenEditModalButton';
+import { authAtom } from '@/store/atoms';
+import { getCurrentUser } from '@/utils/api/currentUser';
+import { get } from '@api/api_methods';
+import { Card, Title } from '@components/common';
+import MainLayout from '@components/layout/MainLayout/MainLayout';
+import { YearPeriods, User } from '@type/common';
+
+interface Props {
+  yearPeriods: YearPeriods[];
```

Comment:
単数系にすることで、yearPeriodsはYearPeriod型の配列なんだなと意味が通りやすくなります

```suggestion
  yearPeriods: YearPeriod[];
```

YearPeriods[]だと年度が複数入ったものの配列（2次元配列）なんだなと勘違いしてしまう可能性があります

Reply:
修正しました [[fix]type変更に伴う年度一覧ページ修正](https://github.com/NUTFes/FinanSu/pull/660/commits/4e4e0d002abb90ca58a57f6b80894ade19be8473)